_studio/shared/include/mfx_trace.h

```diff
@@ -404,6 +404,33 @@ mfxTraceU32 MFXTrace_EndTask(mfxTraceStaticHandle *static_handle,
 
 #ifdef __cplusplus
 
+#if defined(MFX_TRACE_ENABLE_CHROME) && (defined(LINUX32))
+#include <stdint.h>
+#include <vector>
+#include <string>
+
+class ChromeTraceEvent {
+private:
+    const char *name, *tid;
+    uint64_t id;
+    std::vector <std::pair <std::string, std::string> > info;
+public:
+    void add_info(const std::string &key, const std::string &value);
+    void add_info(const std::string &key, void *value);
+    ChromeTraceEvent(const char *name, const char *tid);
+    ~ChromeTraceEvent();
+};
+
+#define MFX_TRACE_CHROME(name, type) \
+    ChromeTraceEvent chrome_trace_event_##name(#name, #type)
```

Comment:
well, all that concept is already implemented in mfx_trace, but instead of integrating into existing functionality being a normal trace module you implement yet-another-one tracing module with separate settings, support, etc. I am against that. Basically, if you don't like something in the existing trace module - this needs to be fixed per some plan rather than implementing something new.